zot/image_dest.go

```diff
@@ -74,16 +74,17 @@ func (o *zotImageDest) PutBlob(ctx context.Context, stream io.Reader, inputInfo
 		}
 	}
 
-	fmt.Printf("calling startlayer\n")
 	// Do this as a chunked upload so we can calculate the digest, since
 	// caller is not giving it to us.
 	path, err := o.s.StartLayer()
-	fmt.Printf("callled startlayer and got %s %v\n", path, err)
 	if err != nil {
+		fmt.Errorf("called StartLayer and got path(%s) and error(%v)", path, err)
```

Comment:
This looks like a no-op, as does the one below?

Reply:
The idea was to keep the logs in the error path (assuming zot could misbehave and we are able to debug)

Reply:
Right, but fmt.Errorf() by itself does nothing; you need to assign the result somewhere or print it, right?

Reply:
ah, meant to use logrus, that is what copy/copy.go uses